javascript/organizing-js/factory-functions.md

```diff
@@ -250,22 +250,21 @@ Modules are actually very similar to factory functions. The main difference is h
 Meet a module:
 
 ~~~javascript
-const calculator = (() => {
-  const add = (a, b) => a + b;
-  const sub = (a, b) => a - b;
-  const mul = (a, b) => a * b;
-  const div = (a, b) => a / b;
-  return {
-    add,
-    sub,
-    mul,
-    div,
-  };
-})();
+var myModule = (function() {
```

Comment:
We probably don’t want to use var here, and we should probably just extended the calculator instead of using an abstract example.

Reply:
I can change the vars, but I don't know what you would have in mind for the calculator